doc/plugin/DynamicMacros.md

```diff
@@ -0,0 +1,96 @@
+# Kaleidoscope-DynamicMacros
+
+Dynamic macros are similar to [Macros][plugin:macros], but unlike them, they can
+be re-defined without compiling and flashing new firmware: one can change
+dynamic macros via [Focus][plugin:focus], using a tool like
+[Chrysalis][chrysalis].
+
+ [plugin:macros]: Macros.md
+ [plugin:focus]: FocusSerial.md
+ [chrysalis]: https://github.com/keyboardio/Chrysalis
+
+Dynamic macros come with certain limitations, however: unlike the built-in
+macros, dynamic ones do not support running custom code, they can only play back
+a sequence of key events, and do so whenever one presses the dynamic macro key.
```

Comment:
Is it only -key- events? or do they also support mouse clicks / mousekey events?

Reply:
You are correct, the wording here isn't clear: any `Key` is supported. Depending on the step, of course, some only support keys, but the main variants support mouse clicks, mousekeys, and everything else too. Same thing as for the `MACRO()` helper in `Macros`.

Reply:
fix the docs, then I think this is mergeable